pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -1817,31 +1817,26 @@ func (s *AWSCloud) listSubnetIDsinVPC(vpcId string) ([]string, error) {
 }
 
 // EnsureLoadBalancer implements LoadBalancer.EnsureLoadBalancer
-// TODO(justinsb) It is weird that these take a region.  I suspect it won't work cross-region anyway.
-func (s *AWSCloud) EnsureLoadBalancer(name, region string, publicIP net.IP, ports []*api.ServicePort, hosts []string, serviceName types.NamespacedName, affinity api.ServiceAffinity) (*api.LoadBalancerStatus, error) {
-	glog.V(2).Infof("EnsureLoadBalancer(%v, %v, %v, %v, %v, %v)", name, region, publicIP, ports, hosts, serviceName)
+func (s *AWSCloud) EnsureLoadBalancer(service *api.Service, hosts []string) (*api.LoadBalancerStatus, error) {
+	glog.V(2).Infof("EnsureLoadBalancer(%v, %v)", *service, hosts)
```

Comment:
This is actually going to be significantly LESS useful as a log.  We don't generally have String() methods for API types, so this will be much harder to keep track of.